examples/functions/fares/rail/Netex_era_distance_ro.xml

```diff
@@ -8787,8 +8787,8 @@ State Secretariat for Revolutionary Issues in December 1989: : 12 free roun
 																		<CompanionProfileRef version="any" ref="tfc:TFC@Tariff@General@eligibilities@veteran@companion@personal_assistant"/>
 																	</limitations>
 																	<validityParameters>
-																		<DatedVehicleJourneyRef ref="tfc:train-IR1950" versionRef="External">Specific journey for reservation</DatedVehicleJourneyRef>
 																		<ClassOfUseRef ref="eura:first" version="any"/>
+																		<ServiceJourneyRef ref="tfc:train-IR1950" versionRef="External">Specific journey for reservation</ServiceJourneyRef>
```

Comment:
And this did not come from xmllint, or isn't the folder included or something like it?

Reply:
@skinkie I don't know why it did not scan those.